usaspending_api/awards/tests.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase, Client
+
+class AwardTests(TestCase):
+
+    #load the test fixture
+    fixtures = ['awards']
```

Comment:
Super minor, but I've never been a fan of loading fixtures from a file -- it requires the test readers look in multiple places to understand what's being tested. 

Out of scope for this PR, but something I'd recommend looking at: the sooner you get `py.test` in, the sooner you'll see it's benefits. It's completely compatible with unittest-style stuff, but I don't think you'll want to go back.